development/tools/pkg/release/options.go

```diff
@@ -36,21 +32,14 @@ func NewOptions(ctx context.Context, storage StorageAPI, releaseVersionFilePath,
 
 	//Changelog
 	var releaseChangelogData string
-	if _, err := os.Stat(releaseChangelogName); err != nil {
-		if !os.IsNotExist(err) {
-			return nil, errors.Wrapf(err, "while reading %s file", releaseChangelogName)
-		}
-		// use try to use GCP to fetch the changelog
-		releaseChangelogData, err = relOpts.readReleaseBody(ctx, releaseVersion, releaseChangelogName)
-		if err != nil {
-			return nil, errors.Wrapf(err, "while reading %s file", releaseChangelogName)
-		}
-	} else {
+	if _, err := os.Stat(releaseChangelogName); err == nil {
 		clb, err := os.ReadFile(releaseChangelogName)
 		if err != nil {
 			return nil, errors.Wrapf(err, "while reading %s file", releaseChangelogName)
 		}
 		releaseChangelogData = string(clb)
+	} else {
+		return nil, errors.Wrapf(err, "while reading %s file", releaseChangelogName)
 	}
```

Comment:
Usually when you write the statements like this, do not check if err==nil, but fail and exit if the err != nil. The you don't need else statement